scripts/upload-releasenotes.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+BRANCH="$(git symbolic-ref --short HEAD)"
+NOTES=${1:-release-notes.txt}
```

Comment:
```suggestion
NOTES=${1:-releasenotes.txt}
```

Reply:
`release-notes.txt` is what we have used most of the time. I can change it to releasenotes.txt though.